sorc/emcsfc_snow2mdl.fd/grib_utils.f

```diff
@@ -2,7 +2,7 @@
 !! @brief Determine whether file is grib or not.
 !
 !> Determine whether file is grib or not.
-!! @author gayno org: w/np2 @date 2007-nov-28
+!! @author gayno org: w/np2 @date 2007-nov-28   #GWV check for action
```

Comment:
Remove: #GWV check for action
Change to George's full name. 
If we're going to spell "nov" then capitalize it.